### PR TITLE
add to development Graphql-playground-middleware-express

### PR DESCRIPTION
**Pull request by Rohcode**
# Description
Adding development GraphQL Playground.

### CODE REVIEW

File: examples/neo-push/server/src/server.ts [hunk 1]
```diff
@@ -19,6 +19,15 @@ if (config.NODE_ENV === "production") {
     });
 }
 
+if (config.NODE_ENV === "development"){
+    debug("Development serving graphql-playground");
+    const expressPlayground = require('graphql-playground-middleware-express').default
```
Comment (danstarns) [inline examples/neo-push/server/src/server.ts:24]: Could we use the `import` syntax here to be consistent with the rest of the codebase?

File: examples/neo-push/server/package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
         "apollo-server-testing": "2.19.0",
         "cross-env": "7.0.3",
         "faker": "5.1.0",
+        "graphql-playground-middleware-express": "^1.7.22",
```
Comment (danstarns) [inline examples/neo-push/server/package.json:39]: Please could you pin this: 

```
"graphql-playground-middleware-express": "1.7.22",
```

Pining the dependencies means we reduce the chance of things breaking on version increments.

**Review by danstarns — CHANGES_REQUESTED** [review]
Thank you for this, looks good and works just fine. Two comments I added and just to point out you will need to sign our CLA agreement http://neo4j.com/developer/cla

**Review by Rohcode — COMMENTED** [review]
Yep

**Comment by oskarhane** [comment]
@Rohcode Hi, I can't find your CLA signature. Did you send it to us?
http://neo4j.com/developer/cla

**Comment by Rohcode** [comment]
Hi. My name is Rohan Thompson ***@***.***, Rohcode)
I agree to the terms in the attached Neo4j Contributor License Agreement with the edit clarifying that contributions are restricted to Neo4j Github projects (see attached).

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by oskarhane** [comment]
> Hi. My name is Rohan Thompson ***@***.***, Rohcode) I agree to the terms in the attached Neo4j Contributor License Agreement with the edit clarifying that contributions are restricted to Neo4j Github projects (see attached). Sent with [ProtonMail](https://protonmail.com/) Secure Email. ‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
> […](#)

Sorry for the back and forth with this, but we're going to need you to send the CLA email to the adress specified on http://neo4j.com/developer/cla

The email is in the last section on that page.

**Comment by oskarhane** [comment]
I see that you've signed the CLA now, thanks for that. The next hurdle is that the there's nothing in the committers info that can be tied to your GH user - so our white listing system (that whitelists GH users) don't know that you (your GH user) made the committs.
Could you re-commit using a git user config using the same email address as your GH user @Rohcode?

So sorry for the inconvenience.

**Comment by Rohcode** [comment]
Sorry I'm confused...
I used @Rohcode (my GH account tied to this email) to make the neo-push pull request...Thought that was how you got this email.. It was a small commit, just integrated GraphQL playground for express..

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by darrellwarde** [comment]
Closing due to inactivity and the fact that commits were not made using git configured with the same details as user's GitHub account.

**Pull request closed without merging**